content/sdk/10.js/00.ethers/10.v5/00.getting-started.md

```diff
@@ -12,3 +12,126 @@ ensuring smoother and more efficient workflows.
+  const result = await provider.getTransactionDetails(transactionHash);
+  console.log(`Transaction Details: ${JSON.stringify(result)}`);
+})();
+```
+
+## Error handling
+
+Wrap the main logic in a try-catch block to handle potential errors:
+
+```typescript
+(async () => {
+  try {
+    // Your code here
+  } catch (error) {
+    console.error(`Error fetching transaction details from ZKsync: ${error}`);
+  }
+})();
+```
+
+## Example code
+
+Here is an example script demonstrating the usage of the SDK:
+
+```typescript
+import { Provider, types } from "zksync-ethers";
+
+async function main() {
+  const provider = Provider.getDefaultProvider(types.Network.Sepolia);
+
+  const network = await provider.getNetwork();
+  const blockNumber = await provider.getBlockNumber();
+  console.log(`Network: ${JSON.stringify(network)}`);
+  console.log(`Block number: ${blockNumber}`);
+
+  const transactionHash = "0xaa065e5a57e1f8470a6f258e2b6eee87c547eab066b8620ce7f3fd51405665e1";
+
+  try {
+    const result = await provider.getTransactionDetails(transactionHash);
+    if (result) {
+      console.log(`Transaction Details from ZKsync: ${JSON.stringify(result)}`);
+      console.log(`ethCommitTxHash: ${result.ethCommitTxHash}`);
+      console.log(`ethExecuteTxHash: ${result.ethExecuteTxHash}`);
+      console.log(`ethProveTxHash: ${result.ethProveTxHash}`);
+    } else {
+      console.log(`Transaction with hash ${transactionHash} not found in ZKsync.`);
+    }
+  } catch (error) {
+    console.error(`Error fetching transaction details from ZKsync: ${error}`);
+  }
+}
+
+main()
+  .then(() => console.log("Script executed successfully"))
+  .catch((error) => console.error(`Error executing script: ${error}`));
+```
+
+## Run the script
+
+Use the following command to run the script:
+
+```bash
+npx ts-node <file-name>
+```
+
+## Output
+
+Example output when the script runs successfully:
+
+```sh
+Network: {"name":"unknown","chainId":"300"}
+Block number: 2725282
+Transaction Details from ZKsync: {"ethCommitTxHash":"0xfe921b3af6bf14d35d6c550f1a337f20a46997a36c24bae37c1b2d129ee3b4d6","ethExecuteTxHash":"0x08e42763d6ba1052d117174acbb708d9e015ae9246574cf9d9b06c001b31e750","ethProveTxHash":"0x49ab3b1a7cc72911492afe39f21d5f557abcb2769aa63841f658c719a7ec5ba2","fee":"0x1252b3c112d2e","gasPerPubdata":"0xc350","initiatorAddress":"0xb71ce978bf48e3e4669a7a0acb89850023fc3279","isL1Originated":false,"receivedAt":"2024-06-03T09:16:41.519Z","status":"verified"}
+ethCommitTxHash: 0xfe921b3af6bf14d35d6c550f1a337f20a46997a36c24bae37c1b2d129ee3b4d6
+ethExecuteTxHash: 0x08e42763d6ba1052d117174acbb708d9e015ae9246574cf9d9b06c001b31e750
+ethProveTxHash: 0x49ab3b1a7cc72911492afe39f21d5f557abcb2769aa63841f658c719a7ec5ba2
+Script executed successfully
+```
+
+::callout{icon="i-heroicons-light-bulb"}
+If you encounter any error while using the v5, kindly visit the
+[troubleshooting Guide](/sdk/troubleshooting#javascript-sdk) for more information.
```

Comment:
nit: capitalize 'T' in 'Troubleshooting Guide.'

Reply:
Done. Thanks